base/ubi/9.4-minimal/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1194
```

Comment:
given this builds `base-ubi-9.4-minimal` (rolling) this ref should be rolling as well - it also helps with periodics automatically picking the updates

Reply:
ok